<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Sorter Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QPushButton" name="importFolderButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Import Folder</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="centralHorizontalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="imagePreviewArea">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="imageAreaLayout">
          <item>
           <widget class="QLabel" name="imagePlaceholderLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>640</width>
              <height>360</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>640</width>
              <height>360</height>
             </size>
            </property>
            <property name="text">
             <string>Image Preview</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="imageNavButtonsLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="prevImageButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextImageButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectImageButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="image_label">
            <property name="text">
             <string>Image information</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="folderNavButtonsLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="nextFolderButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Next Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="prevFolderButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>113</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Previous Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="folder_label">
          <property name="text">
           <string>Folder Information</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="sidebarLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="selectedImagesLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Selected Images</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="selectedImagesList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>437</width>
            <height>235</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>437</width>
            <height>235</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>437</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>437</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>437</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>437</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
